lib/ansible/modules/cloud/azure/azure_rm_image_facts.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
```

Comment:
A lot of the same comments I added for `azure_rm_image` in the documentation part also apply here.

Reply:
Done.